src/config/constants.ts

```diff
@@ -68,4 +68,8 @@ export const SKILLS = [
   'Risk',
   'Tokenomics',
   'Contributor Experience',
+  '*People & Governance',
```

Comment:
We want the asterisk at the start of these? Seems different than the other tags? @spydermonkee

Reply:
Yep, @crabsinger - The request specifically included that so that these tags would be easy to stand out. I feel mixed about it, but am more focused on getting this done to empower the Yearn team. If you would like, I could check with the Yearn team and I think the fact that I added them at the end means that they will be easy to find by virtue of being all at the end of the list might be acceptable for "discoverability". Want me to check on that?